build/infrastructure/dh/env/b-002/ui-app-custom-domain.tf

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+resource "azurerm_static_site_custom_domain" "ui_custom_domain" {
+  static_site_id  = azurerm_static_site.ui.id
+  domain_name     = "it-lev.datahub.dk"
```

Comment:
@djorgensendk I think it should be `itlev.datahub.dk`

Reply:
of course. Thanks